patch-capfram.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+cd node_modules/capital-framework/src/cf-core/src
+sed -i '' -e 's|../../|~|g' cf-core.less
```

Comment:
The paths work fine in other projects, though. I'm guessing this has something to do with how the Less compiler is configured in this project. I'll dig a little deeper.